subjects/atm-management-system/audit/README.md

```diff
@@ -1,14 +1,16 @@
 #### Functional
 
+> Note: Confirm the data storage method used in the project (e.g., text files or a relational database like SQLite).
+
 ##### Open the application and register a new user with the name `"Marcus"` and the password `"q1w2e3r4t5y6"`.
 
-###### Is this user saved in the file `"./data/users.txt"`, and if so, are all credentials correct (name and password)?
+###### Is this user saved in the in the data storage (text file `"./data/users.txt"` or database), and if so, are all credentials correct (name and password)?
```

Comment:
`in the` is duplicated here

Reply:
Fixed, did some more small fixes also in the audit